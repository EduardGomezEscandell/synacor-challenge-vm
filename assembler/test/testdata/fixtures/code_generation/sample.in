set r0 'a'
loop:
    add r0 r0 1
    eq r1 r0 'z'
    out r0
    jt r1 loop


