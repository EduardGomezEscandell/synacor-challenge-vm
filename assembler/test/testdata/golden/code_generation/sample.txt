0     1     2     3     4     5     6     7     8     9     a     b     c     d     e     f
01 00 00 80 61 00 09 00 00 80 00 80 01 00 04 00 01 80 00 80 7a 00 13 00 00 80 07 00 01 80 03 00
set   r0    'a'  ;add   r0    r0    1    ;eq    r1    r0    'z'  ;out   r0   ; jt   r1    loop
                  â†‘
                  loop
